docs/first-service.md

```diff
@@ -1,114 +1,127 @@
-            "url": "http://effx.com/help",
-            "label": "Getting Started"
-          },
-          {
-            "url": "https://effxhq.github.io/effx-api-v2/",
-            "label": "API"
-          },
-          {
-            "url": "https://effx.com/help/?shell#cli-documentation",
-            "label": "effx CLI"
-          },
-          {
-            "url": "https://github.com/effxhq/vcs-connect",
-            "label": "vcs-connect"
-          }
-        ]
-      },
-      {
-        "label": "dashboards",
-        "links": [
-          {
-            "url": "http://status.effx.com",
-            "label": "status"
-          },
-          {
-            "url": "https://p.datadoghq.com/sb/f655e3381-6bdce95d9ec6f133df05532b21b5f5dc",
-            "label": "datadog"
-          }
-        ]
-      },
-      {
-        "label": "runbook",
-        "links": [
-          {
-            "url": "https://effx.com/blog/how-to-write-a-runbook",
-            "label": "best practices"
-          }
-        ]
-      },
-      {
-        "label": "version control",
-        "links": [
-          {
-            "url": "https://github.com/effxhq/.github",
-            "label": "github workflows"
-          },
-          {
-            "url": "https://github.com/effxhq/vcs-connect",
-            "label": "vcs-connect"
-          }
-        ]
-      },
-      {
-        "label": "continuous integration",
-        "links": [
-          {
-            "url": "https://effx.com/help/?yaml#using-github-actions",
-            "label": "github actions"
-          },
-          {
-            "url": "https://effx.com/help/?yaml#using-gitlab-ci-cd-jobs",
-            "label": "gitlab"
-          },
-          {
-            "url": "https://circleci.com/developer/orbs/orb/effx/effx-cli",
-            "label": "circleci Orb"
-          }
-        ]
-      }
-    ]
-  }
-}
-'
+
+Setting yourself up for success with effx starts with the microservice catalog.  Services are the base unit of what most effx functionality is built around we've built automation to help you do that easily.
+
+The primary method for importing services into effx is via the GitOps workflow.  _Why?_  It's important that your catalog has all the right data. Peer review is an important part of this process.
+
+effx resources are usually defined in `effx.yaml` files. You will learn how to define them and introduce automated workflows
+to keep your data up-to-date in these docs.
+
+The first step is to generate your first `effx.yaml` file for your initial service and then upload it via one of the methods below:
+
+## Adding your first service
+
+We've created a nifty [in-app YAML wizard](https://app.effx.com/yaml/wizard) to guide you through the process of creating your first service, but if you prefer to write the YAML yourself, below is the full definition of how to construct your own `effx.yaml`.  You can see a [full example](#effx-yaml-example) below.
+
+### effx-yaml definition
```

Comment:
This entire section is redundant with the "Resource Types" section.

I personally found this harder to read than the dynamic attribute component.